### ISSUE TITLE
Cannot create file/solution

### ISSUE THREAD

**Issue by Vftdan** [mono/monodevelop#6020]
When I select File>New File, MonoDevelop doen't react at all (neither Edit>Preferences).
When I try to create a solution, I select project type, press Next, MonoDevelop doesn't react, press Newxt again, MonoDevelop says "Illegal solution name".
![screenshot from 2018-09-17 23-45-23_crop](https://user-images.githubusercontent.com/9785382/45701179-3507b500-bb77-11e8-8758-cfb371cbff94.png)

**Comment by Matthew-Collins** [comment]
Yes, I have the same issue on Linux Mint 19 using my build from source of mono/monodevelop:master.

What version are you running?

**Comment by Matthew-Collins** [comment]
After some testing...steps to reproduce/resolve errors:
1) Install Mint 19
2) Install Mono-Complete (via install page)
3) Install Monodevelop (via install page)
4) Crashes will occur (per screenshot) in Monodevelop at this stage
5) Update gtk-sharp2 via Update Manager (from v2.12.40 to >= 2.12.45)
6) No Crashes

**Comment by knocte** [comment]
Can you reproduce this with Ubuntu18.04? Just curious if Mint<>Ubuntu differences are causing this bug.
